robozonky-strategy-natural/src/main/java/com/github/robozonky/strategy/natural/GeneratedStrategyVerifier.java

```diff
@@ -0,0 +1,19 @@
+package com.github.robozonky.strategy.natural;
+
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+
+import java.io.IOException;
+
+/**
+ * Used for testing of strategies generated by natural-strategy-setup.
+ */
+public class GeneratedStrategyVerifier {
+    public static ParsedStrategy parseWithAntlr(final String strategy) throws IOException {
```

Comment:
@jhrcek Why not reuse the same method that already is in the package? Seems like code duplication.

Reply:
Right, that didn't occur to me. Trying again :)